docs/migration/5_0.md

```diff
@@ -40,3 +40,9 @@ under the section `headerDisplayOnly`.
 #### Translations (i18n) changes
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
+
+### DeliveryModeComponent
+
```

Comment:
- `ng-container` which wrapped `div.row.cx-checkout-btns` has been removed. 
- `div.row.cx-checkout-btns` has been moved to `div.row > div.col-md-12ccol-lg-9 > ng-container`
- `div.row > div.col-md-12.col-lg-9 > ng-container` will appear only if  `supportedDeliveryModes$` is loaded with length and `continueButtonPressed` flag is falsy.